client/config/config.go

```diff
@@ -805,6 +809,10 @@ func DefaultConfig() *Config {
 		CgroupParent:       "nomad.slice", // SETH todo
 		MaxDynamicPort:     structs.DefaultMinDynamicPort,
 		MinDynamicPort:     structs.DefaultMaxDynamicPort,
+		Users: &UsersConfig{
+			MinDynamicUser: 80_000,
+			MaxDynamicUser: 89_999,
+		},
```

Comment:
Just out of curiosity, are these defaults OS specific? I'm not sure how they work on Windows 😬

Reply:
Even worse, 80k-90k happens to be a gap not reserved (by conveention) by systemd. Users of non-systemd linux systems will need to think about this range and whether it is suitable for them. Will be covered in the docs for exec2.